examples/example1.nim

```diff
@@ -1,5 +1,5 @@
-include ../wxCompile
-#import wx
+#include ../wxCompile
+import ../wx
 
```

Comment:
Fair enough, I thought this was just an artifact from testing. Maybe this example should just be removed then since it doesn't really have anything to do with the wxnim package..